staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1/defaults.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	utilpointer "k8s.io/utils/pointer"
+)
+
+func addDefaultingFuncs(scheme *runtime.Scheme) error {
```

Comment:
need to add this into the scheme at https://github.com/kubernetes/kubernetes/blob/79e8a295442a73a14a6f7173e4c1c3b61834d496/staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1/register.go#L46-L51